packages/coil-anonymous-tokens/src/index.ts

```diff
@@ -124,22 +129,23 @@ export class AnonymousTokens {
         await this.populateTokens(coilAuthToken)
         continue
       }
-      const btpToken = await this._redeemToken(token)
-      if (btpToken) return btpToken
+      const redeemedToken = await this._redeemToken(token)
+      if (redeemedToken) return redeemedToken
       // Otherwise, try again, since the retrieved token was likely expired.
     }
   }
 
   private async _redeemToken(
     token: StorableBlindToken
-  ): Promise<string | undefined> {
+  ): Promise<RedeemedToken | undefined> {
     const usableToken = deserializeToken(token)
     const redeemRequest = BuildRedeemHeader(usableToken, '', '')
     const response = await portableFetch(this.redeemerUrl + '/redeem', {
       method: 'POST',
       headers: { 'content-type': 'application/json' },
       body: JSON.stringify({
-        bl_sig_req: redeemRequest
+        bl_sig_req: redeemRequest,
+        mode: 'fast'
```

Comment:
`mode:"fast"` is a new option in the redeemer service -- when `fast` is specified the new BTP token will have refill=0 and capacity=aggregate. This ensures that the token can be spent rapidly within just a few packets, rather than gradually over a full minute.